packages/vscode-extension/esbuild-extension.mts

```diff
@@ -0,0 +1,23 @@
+import * as esbuild from 'esbuild';
+const production = process.argv.includes('--production');
+
+const ctx = await esbuild.context({
```

Comment:
We can do top level await & esm imports in node because we have a modern node version 💯 

The issue with esm importing files in node is that all source files need the `.js` extensions, but we don't import any files here (only dependencies). So we're alright